packages/internal/src/generate/typeDefinitions.ts

```diff
@@ -161,57 +165,81 @@ export const generateTypeDefGlobalContext = () => {
 // and they may want to generate a custom side. :shrug
 // TODO: Figure out how to get a list of scalars from the api-side so that
 // they don't get out of sync.
-export const generateTypeDefGraphQL = async (
-  side: 'web' | 'api' | 'all' = 'all'
-) => {
-  const rwjsPaths = getPaths()
-  type GenerateResponse = { filename: string; contents: string }[]
+export const generateTypeDefGraphQLApi = async () => {
   try {
-    // TODO: Figure out how to get this type.
-    const generates: Record<string, unknown> = {}
-
-    if (['api', 'all'].includes(side)) {
-      generates[path.join(rwjsPaths.api.base, 'types/graphql.d.ts')] = {
-        plugins: ['typescript', 'typescript-resolvers'],
-      }
-    }
-    if (['web', 'all'].includes(side) && findCells().length) {
-      generates[path.join(rwjsPaths.web.base, 'types/graphql.d.ts')] = {
-        documents: './web/src/**/!(*.d).{ts,tsx,js,jsx}',
-        plugins: ['typescript', 'typescript-operations'],
-      }
-    }
-
-    // https://www.graphql-code-generator.com/docs/getting-started/programmatic-usage#using-the-cli-instead-of-core
-    const f: GenerateResponse = await generate(
-      {
-        cwd: rwjsPaths.base,
-        schema: rwjsPaths.generated.schema,
-        config: {
-          scalars: {
-            DateTime: 'string',
-            Date: 'string',
-            JSON: 'Record<string, unknown>',
-            JSONObject: 'Record<string, unknown>',
-            Time: 'string',
+    const rwjsPaths = getPaths()
+    const f = await generateTypeDefGraphQL({
```

Comment:
Hey peter, totally stylistic, but what do you think of naming these vars a little more explicitly? Sometimes its hard to figure out when you've not looked at the code in a while.